package.json

```diff
@@ -8,98 +10,48 @@
   },
   "scripts": {
     "init": "yarn && yarn build",
-    "create-package": "node ./scripts/create-package.js",
+    "create-package": "lg-create",
     "build": "turbo run build tsc",
     "build-storybook": "storybook build",
-    "build:docs": "npx ts-node ./scripts/parse-tsdocs.ts",
-    "lint": "npm-run-all --parallel lint:*",
-    "lint:prettier": "prettier --check \"**/*.{js,ts,tsx,json,md,yml}\"",
-    "lint:eslint": "eslint \"**/*.{js,ts,tsx}\"",
+    "build:docs": "lg-tsdoc",
     "chromatic": "npx chromatic",
     "clean": "yarn clean:builds && yarn clean:modules",
     "clean:builds": "rm -rf packages/*/dist packages/*/tsconfig.tsbuildinfo ./tsconfig.tsbuildinfo packages/**/stories.js",
     "clean:modules": "rm -rf node_modules packages/*/node_modules ",
-    "fix": "npm-run-all --parallel fix:*",
-    "fix:prettier": "prettier --write \"**/*.{js,ts,tsx,json,md,yml}\"",
-    "fix:eslint": "eslint \"**/*.{js,ts,tsx}\" --fix",
-    "link": "./scripts/link.sh",
-    "link:all": "lerna exec -- yarn link",
+    "fix": "lg-lint --fix",
+    "link": "lg-link",
+    "lint": "lg-lint",
```

Comment:
do we want a lint script that doesn't fix?

Reply:
Useful for CI